packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -33,6 +33,7 @@ export class NodeHttpHandler implements HttpHandler {
   private readonly httpsAgent: https.Agent;
   private readonly connectionTimeout?: number;
   private readonly socketTimeout?: number;
+  public readonly metadata = ["h1"];
```

Comment:
Change this to an object instead of an object, and name the key as `handlerProtocol` or something similar

```suggestion
  public readonly metadata = {"handlerProtocol": "h1"};
```

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/567/commits/4c8361d4bd954a3a16e38b984c0610f4cd69612e